common/utils/net.go

```diff
@@ -18,67 +18,35 @@
 package utils
 
 import (
+	"github.com/apache/dubbo-go/common/logger"
 	"net"
```

Comment:
Pls format the import blocks. You can follow others files in dubbo-go.

Reply:
ok i will run go fmt ./...